src/auth/session/test/session.test.ts

```diff
@@ -0,0 +1,50 @@
+import {ApiVersion, ContextParams} from '../../../base_types';
+import {Context} from '../../../context';
+import '../../../test/test_helper';
+import {Session} from '..';
+
+const validContextParams: ContextParams = {
```

Comment:
Valid `Context` is initialized in `text_helper`. Is there a reason we need to create a new `Context` for these tests?

Reply:
Not impossible that this was added before the test_helper case 🤔